frontend/package.json

```diff
@@ -17,6 +17,7 @@
     "@mui/icons-material": "^5.15.14",
     "@mui/material": "^5.15.13",
     "@mui/system": "^5.15.14",
+    "antd": "^5.16.4",
```

Comment:
I think it is better that we remove this and use mui components to ensure consistent ui